components/camel-chatscript/src/main/java/org/apache/camel/component/ChatScriptConsumer.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component;
+
+import java.util.Date;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.apache.camel.impl.ScheduledPollConsumer;
+
+/**
+ * The ChatScript consumer.
+ */
+public class ChatScriptConsumer extends ScheduledPollConsumer {
+    private final ChatScriptEndpoint endpoint;
+    public ChatScriptConsumer(ChatScriptEndpoint endpoint, Processor processor) {
+        super(endpoint, processor);
+        this.endpoint = endpoint;
+    }
+  
+    @Override
+    protected int poll() throws Exception {
+        Exchange exchange = endpoint.createExchange();
+       
+        // create a message body
+        exchange.getIn().setBody( this.endpoint.getCHAT_INIT());
```

Comment:
The same issues again with Java CamelCase :)

Reply:
Just now I have commited the new version. Pleae check.